lib/harmonium.ex

```diff
@@ -196,7 +196,16 @@ defmodule Harmonium do
   Renders an input label.
   """
   def rev_label(modifiers, do: block) do
-    content_tag :label, class: rev_label_class(modifiers) do
+    content_tag :label, class: rev_label_class(class), for: for_attr do
+      block
+    end
```

Comment:
This needs to be reverted since it's causing the build to fail.

Reply:
thanks -- fixed below.